mkversion.sh

```diff
@@ -130,4 +130,5 @@ EOF
 
 cat <<EOF > "$PKG_BUILDDIR/data/info"
 VERSION=$v
+SNAPD_APPARMOR_REEXEC=0
```

Comment:
Hmm I'm almost ready to suggest adding SNAP_MOUNT_DIR at some point to have a reasonable fallback for incorrectly guessed system specific paths.

Reply:
that's an interesting idea